connect/models/product.py

```diff
@@ -27,16 +28,23 @@ def make_object(self, data):
 class Item(BaseModel):
     global_id = None  # type: str
     mpn = None  # type: str
-    old_quantity = None  # type: int
-    quantity = None  # type: int
+    old_quantity = None  # type: Optional[int]
+    quantity = None  # type: Optional[int]
 
 
 class ItemSchema(BaseSchema):
     global_id = fields.Str()
     mpn = fields.Str()
-    old_quantity = fields.Integer()
-    quantity = fields.Integer()
+    old_quantity = fields.Str()
+    quantity = fields.Str()
```

Comment:
Ok, in all the products we are working on, the quantity was reported as either 'unlimited' or a string-encoded number, not as an int directly. I talked to Marc and he said it could also be a float. I am going to write a custom field for Marshmallow that will accept int, float or str in the schema, and provide and int or float as value ('unlimited' will produce -1, a string encoded number will produce int or float).